docs/ERC721.md

```diff
@@ -248,7 +248,7 @@ The ERC721_Pausable preset creates a contract with pausable token transfers, min
 
 The ERC721_Metadata extension allows your smart contract to be interrogated for its name and for details about the assets which your NFTs represent.
 
-We follow OpenZeppelin's Solidity approach of integrating the Metadata methods `name`, `symbol`, and `tokenURI` into all ERC721 implementations. If preferred, a frontend contract can be created that imports from the ERC721_base directly (and does NOT include the Metadata extension). 
+We follow OpenZeppelin's Solidity approach of integrating the Metadata methods `name`, `symbol`, and `tokenURI` into all ERC721 implementations. If preferred, a contract can be created that does not import the Metadata methods from the ERC721_base library. Note that the IERC721_Metadata interface id should be removed from the constructor as well. 
```

Comment:
These should be code-formatted for consistency. I believe there's a couple extra missing ones.
```suggestion
We follow OpenZeppelin's Solidity approach of integrating the Metadata methods `name`, `symbol`, and `tokenURI` into all ERC721 implementations. If preferred, a contract can be created that does not import the Metadata methods from the `ERC721_base` library. Note that the `IERC721_Metadata` interface id should be removed from the constructor as well. 
```

Reply:
Probably also mention that all presets are _Mintable_. They are, right?